db/constants.conf

```diff
@@ -434,6 +434,9 @@ constants_db: {
 	cell_landprotector: 5
 	cell_novending:     6
 	cell_nochat:        7
+	//cell_icewall:       8
```

Comment:
why this constants commented?

Reply:
they weren't there before, iassumed there was a reason for it? i didnt know what to do with them so i just added them as comments.